plugins/travis-ci/src/state/useBuildWithSteps.ts

```diff
@@ -37,7 +37,7 @@ export function useBuildWithSteps(buildId: number) {
         vcs: {
           owner: owner,
           repo: repo,
-          type: GitType.GITHUB,
+          type: 'github',
```

Comment:
I noticed that Travis CI supports BitBucket in beta so it would make sense to support that too.